packages/cli/internal/pkg/cli/workflow/manager.go

```diff
@@ -348,6 +361,35 @@ func (m *Manager) parseInputToArguments() {
 	m.arguments = []string{arguments}
 }
 
+func (m *Manager) parseAndAddToManifest() {
```

Comment:
I would rename this to `writeTempManifest` to increase readability.

Reply:
I know our code is barely documented, you should take a quick look at https://go.dev/doc/comment and add godoc for the newly introduced code by u to increase readability.